bedrock/firefox/templates/firefox/whatsnew/index-lite.id.html

```diff
@@ -31,16 +31,16 @@ <h1 class="c-page-header-up-to-date">{{ _('Your Firefox is up to date.') }}</h1>
   </header>
   <div class="lite-body">
     <div class="lite-content mzp-l-content">
-      <div class="lite-woman">
-        <img src="{{ static('img/firefox/whatsnew/lite/woman.png') }}" width="513" alt="">
+      <div class="lite-KV">
+        <img src="{{ static('img/firefox/whatsnew/lite/phone.png') }}" alt="">
       </div>
       <div class="lite-copy">
         <div class="lite-container">
-          <img src="{{ static('img/logos/firefox/logo-lite.png') }}" width="100" alt="">
-          <h1><img src="{{ static('img/logos/firefox/logo-lite-wordmark-white.png') }}" width="300" alt="Firefox Lite"></h1>
+          <img src="{{ static('img/logos/firefox/logo-lite-2019.png') }}"  alt="" height="146" width="155">
+          <h1><img src="{{ static('img/logos/firefox/logo-lite-wordmark-dark.png') }}"  alt="The New Firefox Lite"></h1>
```

Comment:
Official word from Brand is that it shouldn't be the wordmark if it appears in a sentence like this. We can probably make this a regular text title, and I assume it would use the Metropolis font like other Firefox branded pages.

Also got confirmation the product name is "Firefox Lite." Quoth Tim:

> Firefox Lite operates independently of the "Firefox Browser" construct, just as "Firefox Reality" does. "Lite" is the name of this browser, which is a substantially different type of browser than the build for the Firefox Browser products.